gateway/api_definition.go

```diff
@@ -334,22 +351,13 @@ func (a APIDefinitionLoader) MakeSpec(def *nestedApiDefinition, logger *logrus.E
 
 	spec.GlobalConfig = a.Gw.GetConfig()
 
-	// Create and init the virtual Machine
-	if a.Gw.GetConfig().EnableJSVM {
-		mwPaths, _, _, _, _, _, _ := a.Gw.loadCustomMiddleware(spec)
-
-		hasVirtualEndpoint := false
-
-		for _, version := range spec.VersionData.Versions {
-			if len(version.ExtendedPaths.Virtual) > 0 {
-				hasVirtualEndpoint = true
-				break
-			}
-		}
+	if err = a.Gw.loadBundle(spec); err != nil {
```

Comment:
loading bundle during MakeSpec so that loader is aware of driver mentioned in bundle. 
a network call would be made only if bundle isn't found locally, so there's no much overhead there